pageserver/src/tenant.rs

```diff
@@ -1119,11 +1119,13 @@ impl Tenant {
     /// If the caller specified the timeline ID to use (`new_timeline_id`), and timeline with
     /// the same timeline ID already exists, returns None. If `new_timeline_id` is not given,
     /// a new unique ID is generated.
+    ///
+    /// `ancestor_start_lsn` should always be specified, because there is a race between branch
+    /// creation and updating the ancestor. The caller should take care of it.
```

Comment:
There is no `ancestor_start_lsn` anymore

```suggestion
    /// If `ancestor` is specified, the new timeline is branched off from `ancestor.timeline_id`at
    /// `ancestor.start_lsn`. Otherwise, a new "root" timeline is created, by running
    /// `initdb`.
```